atom/browser/native_window_mac.mm

```diff
@@ -1135,6 +1135,14 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
   return [window_ attachedSheet] == nil;
 }
 
+void NativeWindowMac::SetEnabled(bool enable) {
+  if (enable == false){
+    [window_ attachedSheet] = nil;
+  } else {
+    [window_ beginSheet:window_];
+  }
```

Comment:
_(minor)_ this is non-idiomatic for two reasons, one is comparing a bool to `true` or `false`, the other is all other things being equal, prefer positive conditionals rather than negative as they're generally easier to read.

So, slightly better:

```objc
+void NativeWindowMac::SetEnabled(bool enable) {
￼+  if (enable){
+    [window_ beginSheet:window_];
+  } else {
+    [window_ attachedSheet] = nil;
+  }